backend/src/main/scala/cromwell/backend/wdl/WriteFunctions.scala

```diff
@@ -15,12 +16,10 @@ trait WriteFunctions extends WdlStandardLibraryFunctions {
 
   private lazy val _writeDirectory = writeDirectory.createPermissionedDirectories()
 
-  override def writeFile(path: String, content: String): Try[WdlFile] = {
+  override def writeFile(path: String, content: String): Future[WdlFile] = {
     val file = _writeDirectory / path
-    Try {
-      if (file.notExists) file.write(content)
-    } map { _ =>
-      WdlFile(file.pathAsString)
-    }
+    Future.fromTry(
+      Try(if (file.notExists) file.write(content)) map { _ => WdlFile(file.pathAsString) }
```

Comment:
ToL: Is there no `Future` based `write` method on `better.files`?

Reply:
better file only wraps nio that is entirely synchronous so no :/